CMakeLists.txt

```diff
@@ -278,11 +278,10 @@ configure_file (
   "${PROJECT_SOURCE_DIR}/version.h"
   )
 
-configure_file (
-  "${PROJECT_SOURCE_DIR}/tcmu.conf_install.cmake.in"
-  "${PROJECT_SOURCE_DIR}/tcmu.conf_install.cmake"
-  )
-install(SCRIPT tcmu.conf_install.cmake)
+if (EXISTS /etc/tcmu/tcmu.conf)
+	file(RENAME /etc/tcmu/tcmu.conf /etc/tcmu/tcmu.conf.old)
```

Comment:
Maybe we should use the timestamp instead of old:

+ string(TIMESTAMP newname "tcmu.conf-%Y-%m-%dT%H:%M:%S" UTC)                                                                                                            + file(RENAME /etc/tcmu/tcmu.conf /etc/tcmu/${newname})

Reply:
$ cmake . && make install -> will get a new time stamp and works as we expect.
$ make install -> just make install without cmake, will always use old timestamp generated during cmake, and cause override of old original timestamp file with new tcmu.conf